docker-entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ -z "$SECRET_KEY" ]; then
+	echo "WARNING: No SECRET_KEY set - generating radom key..."
+  export SECRET_KEY="$(tr -dc 'a-z0-9!@#$%^&*(-_=+)' < /dev/urandom | head -c50)"
```

Comment:
This is really interesting, but you might encounter issues very hard to debug if you keep the same database and access it with different SECRET_KEYS setup.

Does it generate different keys if you run twice the same image?

Reply:
From my testing, it seams like we generate a new key on every run.

Reply:
I would recommand writing an error message and steps to generate the secret key and exiting there.

Reply:
See https://github.com/niwo/seven23_server/pull/1

Reply:
Yes, the SECRET_KEY is always randomly generated when there is non defined.
This could be intended when you are testing and want to start with a fresh install, but clearly not for production where you want to define your SECRET_KEY in your ENV.

Reply:
Even for testing it is a bit fragile.